src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignment_submissions/test_lmsassignment_submissions.py

```diff
@@ -20,7 +20,7 @@
 from tests_integration_sql.orchestrator import run_harmonizer
 
 
-SOURCE_SYSTEM = "Test_LMS"
+SOURCE_SYSTEM = "Canvas"
```

Comment:
Now that there are separate blocks of code for each LMS, can you add explicit tests for each of three, and a fourth scenario where the source system is unknown (and thus won't be harmonized)?

Reply:
Done